python/salpytools/states.py

```diff
@@ -74,13 +74,11 @@
 state_matrix[3][4] = True
 state_matrix[5][1] = True
 
-# Set up same state transitions as allowed 
+# Set up same state transitions as allowed
```

Comment:
See above; these are now disallowed.

Reply:
This is part of another ticket: 
https://jira.lsstcorp.org/browse/DM-15818